app/controllers/admin/whitelists/organisation_names_controller.rb

```diff
@@ -22,12 +22,8 @@ def create
   def destroy
     custom_org = CustomOrganisationName.find(params.fetch(:id))
 
-    if custom_org.destroy
-      notice = "Successfully removed #{custom_org.name}"
-    else
-      redirect_to admin_whitelist_organisation_names_path, notice: custom_org.errors.full_messages
-    end
-    redirect_to admin_whitelist_organisation_names_path, notice: notice
+    custom_org.destroy
+    redirect_to admin_custom_organisations_path, notice: "Successfully removed #{custom_org.name}"
```

Comment:
This model has no relations.
I cannot see a scenario where deleting one would ever raise an error.

Showed up as untested code in SimpleCov.